src/Agent.Plugins/PipelineCache/PipelineCacheTaskPluginBase.cs

```diff
@@ -93,16 +93,16 @@ public async Task RunAsync(AgentTaskPluginExecutionContext context, Cancellation
                 context.Warning(OldKeyFormatMessage);
             }
 
-            context.Output($"Resolving key `{string.Join(" | ", keySegments)}`...");
+            context.Output($"Resolving key: {string.Join(" | ", keySegments)}");
             Fingerprint keyFp = FingerprintCreator.EvaluateKeyToFingerprint(context, worksapceRoot, keySegments);
-            context.Output($"Resolved to `{keyFp}`.");
+            context.Debug($"Resolved to `{keyFp}`.");
```

Comment:
Thinking if knowing the hash for every key is useful to show...

Reply:
This is just a more cryptic (pipe-delimited) version of what we're already showing. It wouldn't show the hash of static string either, so it's really not adding much value, but I kept it because it is showing the complete fingerprint value that gets passed to the server.

Reply:
I reevaluated and switched this back to `context.Output` ..